keras_nlp/models/opt/opt_backbone.py

```diff
@@ -172,3 +174,71 @@ def token_embedding(self):
     @classproperty
     def presets(cls):
         return copy.deepcopy(backbone_presets)
+
+    @classmethod
```

Comment:
I don't see `cls` used. Can this be a static method instead?

Reply:
For backbone this could be a static method, for the task, we actually use cls to read the backbone cls. I just decided to keep `classmethod` in all cases for simplicity.